pkg/pod/entrypoint.go

```diff
@@ -19,6 +19,7 @@ package pod
 import (
 	"errors"
 	"fmt"
+	"github.com/tektoncd/pipeline/pkg/version"
```

Comment:
this needs to be with the other `github.com/…` imports (`goimports` should do that automatically)